config/samples/resources/edgecontainercluster/edgecontainercluster_v1beta1_controlplane.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: edgecontainer.cnrm.cloud.google.com/v1beta1
+kind: EdgecontainerCluster
+metadata:
+  name: edgecontainercluster-remote-control-plane
+  labels:
+    label-one: "value-one"
+spec:
+  projectRef:
+    external: ${PROJECT_ID}
```

Comment:
nit: recommend add a comment for those variables like `# Replace ${PROJECT_ID?} with your Google Cloud project id`. Same comment applies to the other files.

Reply:
Done